BaseApp/COD3007.TXT

```diff
@@ -2,8 +2,9 @@ OBJECT Codeunit 3007 DotNet_String
 {
   OBJECT-PROPERTIES
   {
-    Date=;
-    Time=;
+    Date=23.07.18;
```

Comment:
Try to avoid unnecessary changes which distract from the code you'd like us to review, such as adding date/time or the LineStart below; both of which shouldn't be submitted to the repo ;-)